nipype/interfaces/base.py

```diff
@@ -31,6 +31,7 @@
 from warnings import warn
 import simplejson as json
 from dateutil.parser import parse as parseutc
+from packaging.version import Version
 
 from .. import config, logging, LooseVersion, __version__
```

Comment:
It's used for versioning on external programs, e.g. FSVersion < 6.0 or FSLVersion > 507. Since these don't follow PEP440, I intended to leave them as they are.